app/npm-shrinkwrap.json

```diff
@@ -1,1139 +1,928 @@
 {
   "name": "desktop",
   "version": "0.6.1",
+  "lockfileVersion": 1,
   "dependencies": {
     "7zip": {
-      "version": "0.0.6",
-      "from": "7zip@0.0.6",
-      "resolved": "https://registry.npmjs.org/7zip/-/7zip-0.0.6.tgz",
+      "version": "https://registry.npmjs.org/7zip/-/7zip-0.0.6.tgz",
```

Comment:
Looks like you're running `npm@5`, which we've not tested thoroughly. I fear changes like this `version` field are going to cause us issues down the track.